docs/docs/guides/tools/saved-views.md

```
@@ -0,0 +1,173 @@
+   - Column pinning
+3. In the upper right hand corner, click **Save view**. 
+
+### Load a saved view
+
+1. Navigate to your **Traces** or **Evals** tab.
+2. Click the hamburger menu to the left of the tab title. A dropdown menu showing all saved views displays. 
+3. Click the view that you want to access. The saved view displays in the **Traces** or **Evals** tab. 
+
+### Rename a saved view
+
+1. Follow the steps described in [Access a saved view](#access-a-saved-view).
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)